lib/taskana-core/src/main/java/pro/taskana/task/internal/ServiceLevelHandler.java

```diff
@@ -48,14 +46,9 @@
     this.taskanaEngine = taskanaEngine;
     this.taskMapper = taskMapper;
     this.attachmentMapper = attachmentMapper;
-    DaysToWorkingDaysConverter.setGermanPublicHolidaysEnabled(
-        taskanaEngine.getEngine().getConfiguration().isGermanPublicHolidaysEnabled());
-    try {
-      converter = DaysToWorkingDaysConverter.initialize();
-    } catch (InvalidArgumentException e) {
-      LOGGER.error(ERROR_CANNOT_INITIALIZE_DAYS_TO_WORKING_DAYS_CONVERTER);
-      throw new SystemException(
-          ERROR_CANNOT_INITIALIZE_DAYS_TO_WORKING_DAYS_CONVERTER, e.getCause());
+    this.converter = WorkingDaysToDaysConverter.initialize();
+    if (taskanaEngine.getEngine().getConfiguration().isGermanPublicHolidaysEnabled()) {
```

Comment:
Why don't we provide the converter on engine level and initialize it during startup according to the configuration?

Reply:
Apart from the DaysToWorkingDaysReportConverter, the converter is only needed in ServiceLevelHandler. I think, we should maintain and supply classes / objects there where they are needed. But if you insist to provide it on engine level, I can do that.
With respect to configuration: I introduce the configuration property taskana.german.holidays.enabled and set it to true in taskana.properties. Then I initialize this property in TaskanaEngineConfiguration and use it in the constructor of TaskanaEngineImpl to set that (static) property on WorkingDaysToDaysConverter.